src/V3Tristate.cpp

```diff
@@ -914,6 +914,44 @@ class TristateVisitor final : public TristateBaseVisitor {
                || (strength1 <= strength && assignmentOfValueOnAllBits(assignp, 1));
     }
 
+    void mergeWiredNetsAssignments() {
+        // Support for WOR/TRIOR/WAND/TRIAND, by merging the Assignments for the
+        // same Net (merge by or for WOR/TIOR and merge by and for WAND/TRIAND).
+        for (auto& varpAssigns : m_assigns) {
+            Assigns& assigns = varpAssigns.second;
+            if (assigns.size() > 1) {
+                AstVar* varp = varpAssigns.first;
+                if (varp->isWiredNet()) {
+                    auto it = assigns.begin();
+                    AstAssignW* assignWp0 = *it;
+                    FileLine* const fl = assignWp0->fileline();
+                    AstNodeExpr *wExp = nullptr;
+                    const auto getMergeExpr = [&](AstNodeExpr* lhsp, AstNodeExpr* rhsp) {
```

Comment:
Please make separate function instead of lambda.

```suggestion
                    AstNodeExpr* newMergeExpr(AstNodeExpr* lhsp, AstNodeExpr* rhsp);
```